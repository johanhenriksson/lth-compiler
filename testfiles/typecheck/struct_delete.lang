struct str { }

void main() {
    str s;
    delete s; // ok

    int a;
    delete a; // type error
}
