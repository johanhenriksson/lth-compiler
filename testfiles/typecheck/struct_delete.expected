Error at line 8,5: expected delete operand to be of type struct, got int
